jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
nope. It fails for a reason. I didn't write tests, because I was building a prototype, but if people want to use this in production it should have 100 coverage, otherwise each next release can break anything. I explicitly said there are no tests in introduction, there is a ticket for tests (with need help label), there is a red badge of travis in readme.

Reply:
> but if people want to use this in production it should have 100 coverage

Reply:
So far the Travis builds should be successful or show that everything is fine with the library / package, except coverage which we will configure later as needed when we have (more) tests.